### PR TITLE
Beacon chain header relay ( Pangoro<>Kiln )

### PR DESCRIPTION
**Pull request by furoxr**
Beacon chain light client
- Header relay
- Sync committee relay
- Execution payload state root relay

### CODE REVIEW

File: .maintain/config/bridge-pangoro-kiln.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[pangoro]
+endpoint = "https://pangoro-rpc.darwinia.network"
+contract_abi_path = "/path/to/BeaconLightClient_abi.json"
```
Comment (fewensa) [inline .maintain/config/bridge-pangoro-kiln.toml:3]: Bundle contract to code?

Comment (furoxr) [reply .maintain/config/bridge-pangoro-kiln.toml:3]: Emmm, Bundling abi into the code seems better, so useres don't need to find the abi by themselves. I'll update this.

**Comment by fewensa** [comment]
CI failed
https://github.com/darwinia-network/bridger/runs/7312918632?check_suite_focus=true

**Review by HackFisher — APPROVED** [review]

**Review by hujw77 — COMMENTED** [review]
Header relay strategy should be improved.

File: bridges/pangoro-kiln/bridge/src/service/header_relay/kiln_to_pangoro.rs [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+        let old_finalized_header = self.pangoro_client.finalized_header().await?;
+        tracing::info!(
+            target: "pangoro-kiln",
+            "[Header][Kiln => Pangoro] Latest kiln header on pangoro: {:?}, And current kiln header: {:?}",
+            &old_finalized_header.slot,
+            kiln_current_slot,
+        );
+        let _old_period = old_finalized_header.slot.div(32).div(256);
+
+        let attested_header_slot = old_finalized_header.slot.add(96);
```
[the remark this answers is not in the document]
Comment (furoxr) [reply bridges/pangoro-kiln/bridge/src/service/header_relay/kiln_to_pangoro.rs:97]: I'll implement this in next version.

**Pull request merged**